### PR TITLE
HBG: Viconia, Nightsinger's Disciple and support

### LINKED ISSUE #762: Viconia, Nightsinger's Disciple // Specialize

### PR DESCRIPTION
**Pull request by Northmoc**
Only new stuff is MakeCardEffect and Viconia

- closes #762

### CODE REVIEW

**Comment by Northmoc** [comment]
Don't merge until #1330 is merged

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**
